src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/Common/BasicHtmlWebResponseObject.Common.cs

```diff
@@ -57,12 +57,12 @@ public BasicHtmlWebResponseObject(HttpResponseMessage response, Stream contentSt
         #region Properties
 
         /// <summary>
-        /// gets or protected sets the Content property
+        /// Gets or protected sets the Content property.
```

Comment:
Suggest removing 'or protected sets' or change it to 'or sets'. The comment is inaccurate.

Reply:
Fixed.